setup.py

```diff
@@ -73,7 +73,7 @@
     ],
 
     packages=find_packages(exclude=["*.test", "*.test.*", "test.*", "test", "bin"]),
-    package_data={'configs': 'data/configs/*'},
+    package_data={'configs': ['data/configs/*']},
```

Comment:
Actually, I think we can delete this line as the config files don't exist any more.